app/templates/main/login.html

```diff
@@ -0,0 +1,52 @@
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/header/macro.html' import govukHeader -%}
+{%- from 'govuk_frontend_jinja/components/phase-banner/macro.html' import govukPhaseBanner -%}
+{%- from 'govuk_frontend_jinja/components/footer/macro.html' import govukFooter -%}
+
+{% block head %}
+  <meta name="description" content="{{ config['SERVICE_NAME'] }}">
+  <meta name="keywords" content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, jinja, python, flask, port, template, templating, macro, component, design system, html, forms, wtf, wtforms, widget, widgets, demo, example">
+  <meta name="author" content="{{ config['DEPARTMENT_NAME'] }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='govuk-frontend-4.7.0.min.css') }}" />
+  {% assets "css" %}<link href="{{ ASSET_URL }}" rel="stylesheet">{% endassets %}
+{% endblock head %}
+
+{% block header %}
+  {{ govukHeader({
+    'homepageUrl': url_for('main.index'),
+    'serviceName': 'Submit monitoring and evaluation',
+    'serviceUrl': url_for('main.index'),
+  }) }}
+{% endblock header %}
```

Comment:
Could this header be in a macro instead of hardcoded here?

Reply:
i.e. in a base page where we inject the content

Reply:
Yes, but at the moment the way the sign in page is designed and the way that base.html is by default are quite different. We need to change base.html to match the general pattern of the designs, but I think that's probably worthy of another ticket rather than this one?